lineman/app/components/home/home.coffee

```diff
@@ -1,3 +0,0 @@
-angular.module('loomioApp').controller 'HomeController', () ->
```

Comment:
I know it looks like a big PR (it is), but a lot of it is carving out old components we're not using at all.